BuildForSDG/Team-037-Product-backend
# Chore adding swagger

#### What does this PR do?
Tell us what this pull-request you are raising will add to the project.

#### Description of Task to be completed?
list the things you did to complete this project
- install swagger
![Capture50](https://user-images.githubusercontent.com/38535951/82963832-bdc92c00-9fbb-11ea-8d45-e61ce5efb86c.PNG)

- use swagger for a user to be able  to sign up and sign in

#### How should this be manually tested?
Tell us how other people can test out this your feature to see if it's working as expected.
- clone the repo
- run npm install
- go to your browser and run localhost:{Port_number}/docs

#### Any background context you want to provide?
Anything you'll like to say about this pull-request?


#### Screenshots
take pictures of the implementation of the project and paste them here
![Capture51](https://user-images.githubusercontent.com/38535951/82963842-c6b9fd80-9fbb-11ea-9893-817528ca16c7.PNG)
